app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#008577</color>
-    <color name="colorPrimaryDark">#00574B</color>
-    <color name="colorAccent">#D81B60</color>
+    <color name="color_primary">#2e2101</color>
+    <color name="color_primary_dark">#2e2705</color>
+    <color name="color_accent">#D81B60</color>
+    <color name="yellow_light">#656039</color>
```

Comment:
```suggestion
    <color name="color_yellow_light">#656039</color>
```
Em có thể vào link http://chir.ag/projects/name-that-color/#6195ED để check name của color